syntax = "proto3";

package statistics;

service Statistics{

    rpc getSensors (Empty)  returns (stream Sensor);
    rpc getMean (MeanRequest) returns (StringMessage);
}

message Empty{

}

message Sensor {

    int64 _id = 1;
    string data_type = 2;
}

message StringMessage {
    string msg = 1;
}

message MeanRequest {
    int64 _id = 1;
    string data_type = 2;
}